### ISSUE TITLE
Duplicate lines

### ISSUE THREAD

**Issue by nblackburn** [msipos/syntaxic#28]
The Ability to duplicate the selection would be handy, see #27 for potential means of implementation.

**Comment by msipos** [comment]
Agreed, will provide an example plugin implementation in 0.9.3.  Thanks!
